.expeditor/config.yml

```diff
@@ -0,0 +1,39 @@
+# Documentation available at https://expeditor-docs.es.chef.io/
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: chef-notify
+
+github:
+  # The tag format to use (e.g. v1.0.0)
+  version_tag_format: "v{{version}}"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/client-maintainers
+  # Which Github branches to build Omnibus releases from, and what versions
+  # (as determined by the value in the VERSION file) those branches are responsible
+  # for building.
+  release_branch:
+    - master:
+        version_constraint: 14.*
+    - 13-stable:
```

Comment:
You'll need to add a config file to the `13-stable` and `8-stable` branches if you want/need Expeditor to do stuff on those branches.

Reply:
For now I think master only is ok. we can rethink at need